### ISSUE TITLE
Create new "Private Party Paintings" page

### ISSUE THREAD

**Issue by rodneystrong** [EditLLC/client-sublime-time#45]
Create a new page called "Private Party Paintings"

Add all the pictures from this google drive: https://drive.google.com/drive/folders/0B_CwDsLo9u8NfjRTb085dW1yZkg0T0gwdG96ZFRjbkNCZ0QyVjNJelpXUm83UFhxRkk5ejQ

Do it in the same format as this gallery page: http://sublimetimefun.com/gallery.html

You'll have to crop and resize the photos to the correct aspect ratio and then use a larger version of the image for the lightbox. As a rule of thumb, try not to have the larger version be more than 500kb in file size. 

Ping @rampage1213 for a good online image editor. Also, feel free to ask him or @jperez4509 on what the hell I mean by 'aspect ratio' and how it'll pertain to this task. If they don't immediately ELI5 (https://www.reddit.com/r/explainlikeimfive/) it to you, then let me know. 

due:
1-3-16   5p

**Comment by jarantfos** [comment]
I ended up using gimp to resize the images. Also, resizing/cropping all those images was horrible... I think I'm gonna have a Python script ready for next time

**Comment by rodneystrong** [comment]
Heh.. I think you should just create that script right now and push it to a public git repo.
